src/ui-core/templates/pages/HomePage/HomePage.css

```diff
@@ -0,0 +1,3 @@
+.home-page-containerv{
+    
+}
```

Comment:
@SachinthaSampath if this does not have any styles, we can remove it right? Check for any other empty css classes & remove them